core/tests/test_utils.py

```diff
@@ -53,6 +55,9 @@
 
 
 def empty_environ():
+    """Set up empty environment variables for Tests
```

Comment:
Please add a period (.) at the end of the sentence. (Same goes for all the other places)

Reply:
Corrected throughout file.

Reply:
Corrected throughout file.